app/controllers/comments_controller.rb

```diff
@@ -137,14 +137,16 @@ def update
   # DELETE /comments/1.json
   def destroy
     authorize @comment
-    @commentable_path = @comment.commentable.path
     if @comment.is_childless?
       @comment.destroy
     else
       @comment.deleted = true
       @comment.save!
     end
-    redirect_to URI.parse(@commentable_path).path, notice: "Comment was successfully deleted."
+    redirect = @comment.commentable&.path || user_path(current_user)
+    # NOTE: Brakeman doesn't like redirecting to a path, because of a "possible
+    # unprotected redirect". Using URI.parse().path is the recommended workaround.
+    redirect_to URI.parse(redirect).path, notice: "Comment was successfully deleted."
```

Comment:
side note, another one of the oddities in our code:

```shell
➜  devto git:(citizen428/update-foreman) ✗ rg "URI\.parse" | rg -v Addressable | wc -l
      16
➜  devto git:(citizen428/update-foreman) ✗ rg "Addressable::URI\.parse" | wc -l
      11
```

Reply:
I have no strong preference either way. `URI.parse` is core Ruby, `Addressable::URI.parse` come from a gem.

Reply:
No, it's fine :)